packages/doenetml-worker/lib-js-wasm-binding/src/lib.rs

```diff
@@ -122,6 +128,13 @@ impl PublicDoenetMLCore {
         resolver2
     }
 
+    pub fn delete_nodes_from_resolver(resolver: Resolver, node_list: NodeList) -> Resolver {
+        let mut resolver2 = resolver;
```

Comment:
Why is this needed? Can you not do `mut Resolver` in the function signature?

Reply:
Because I couldn't figure out how to do that. I had tried `resolver: mut Resolver`, but it turns out it is `mut resolver: Resolver`.